src/realm/table.hpp

```diff
@@ -394,7 +394,21 @@ class Table {
     /// \sa Table::set_null_unique()
     void merge_rows(size_t row_ndx, size_t new_row_ndx);
 
-    // Get cell values. Will assert if the requested type does not match the column type
+    //@{
+
+    /// Get cell values.
+    /// Will assert if the requested type does not match the column type.
+    ///
+    /// When querying a nullable column and the value is null, a default value
+    /// will be returned, except for object like types (StringData, BinaryData,
+    /// Timestamp) which have support for storing nulls. In that case, call the
+    /// is_null() method on the returned object to check for nullablilty.
+    /// If nullability matters and returning a default value is unacceptable,
+    /// Table::is_null() should be checked before getting a cell value.
```

Comment:
s/call the is_null() method/call the `is_null()` method/

To prevent creation of Doxygen link to `Table::is_null()`.

    s/check for nullablilty/check whether the stored value was null/

Because checking for null-**ability** is a different thing, right? :-)